### DISCUSSION TITLE
How to add flags to logging

### DISCUSSION THREAD

**Discussion by Fregf** [FoobarOy/foomuuri#20] [Q&A] [answered]
I want to log outgoing connections, including the UID of process which made the connection.
So I tried to do what is document on
https://github.com/FoobarOy/foomuuri/wiki/Rule#log_level

I have:
```
localhost-public {
dhcp-client
some-other-allowed-connections
  reject log log_level "level warn flags skuid"
}
```
but it looks like this is interpreted as a prefix to print in the logs:

`Oct 27 14:09:14 hostname kernel: log_level level warn flags skuid IN= OUT=enp1s0 SRC=xxx.xxx.xxx.xxx DST=xxx.xxx.xxx.xxx LEN=60 TOS=0x00 PREC=0x00 TTL=64 ID=20860 DF PROTO=TCP SPT=xxxxx DPT=xxx WINDOW=64240 RES=0x00 SYN URGP=0`

What could be wrong here?

I'm using foomuuri 0.21.1 from Debian Backports on Debian 12.

**Comment by kimheino** [comment, marked as answer]
Sorry for mixup. Per-rule `log_level` was added in da342b8 few days ago. It will be available in upcoming version 0.22.

In 0.21 you can use global option:

```
foomuuri {
  log_level "level warn flags skuid"
}

localhost-public {
  ...
  reject log
}
```

**Comment by Fregf** [comment]
That explains it. Thank you very much!
